_component_design/popovers.md

```diff
@@ -57,23 +60,37 @@ You can use popovers for short create or edit workflows and are particularly use
 
 <section class="static-section" markdown="1">
 
-## Default layout
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+Use these specifications when you construct a popover.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-The most defining aspects of a popover are the container and the pointer. The container appears on top of the canvas. The pointer points to the popover trigger. Pointers can be used on either side of the four corners.
+### Composition
```

Comment:
need a new imagewith a reference bubble for buttons (already exported to png folder)